collective/superfish/upgrades/__init__.py

```diff
@@ -14,3 +13,8 @@ def _cookResources(portal):
 def to_2(context):
     portal = getToolByName(context, 'portal_url').getPortalObject()
     _cookResources(portal)
+
+
+def to_5(context):
+    setup = getToolByName(context, 'portal_setup')
+    setup.runImportStepFromProfile(PROFILE_ID, 'plone.app.registry')
```

Comment:
i personally would copy current registry.xml to the 2_to_5 upgrade profile folder and skip this method
this way the registry settings done in this upgrade step will stay the same. even if registry.xml gets changed in future releases